docs/content/how-to/openstack/create-openstack-cluster.md

```diff
@@ -0,0 +1,281 @@
+# Create a OpenStack cluster
+
+Install an OCP cluster running on VMs within a management OCP cluster
+
+## Limitations
+
+* The HyperShift Operator with OpenStack support is currently in development and is not intended for production use.
+* Ingress and OpenStack CSI (Cinder and Manila) are not functional.
+
+## Prerequisites
+
+* Admin access to an OpenShift cluster (version 4.16+) specified by the `KUBECONFIG` environment variable.
```

Comment:
why 4.16?

Reply:
I took the latest stable version but yeah we can change it.